### ISSUE TITLE
Crashing on IOS after pressing TextInput after Expo Image Picker

### ISSUE THREAD

**Issue by shedworth** [kirillzyusko/react-native-keyboard-controller#155]
**Describe the bug**
In my React Native app, on forms with a text input and Expo image picker, pressing a text input after opening/closing the image picker causes the app to crash. Simply wrapping the app in `<KeyboardProvider>` is enough to cause this behaviour, without using any other components from `react-native-keyboard-controller`. 

**Code snippet**
Note, although this snippet uses `react-hook-forms`, the crashing behaviour happens elsewhere in the app where this isn't used.
```
...
  return (
    <ScrollView
      contentContainerStyle={styles.container}
      style={styles.background}
      showsVerticalScrollIndicator={false}
      >
        <View>
          <Controller
            control={control}
            name={"bannerImage"}
            render={renderBannerImagePicker}
          />
        </View>
      <View style={styles.formSection}>
        <Label
          required
          text={i18n.t("screens.birdingGroupCreateChallenge.name")}
        />
        <Controller name={"name"} control={control} render={renderName} />
      </View>
      </ScrollView>
```

App.tsx
```
...
  return (
    <KeyboardProvider>
      <GestureHandlerRootView style={styles.gestureView}>
        <LoginProvider
          isLoggedIn={isLoggedIn && !!currentUser}
          currentUser={currentUser}
        >
          <DatabaseProvider>
            <ApolloProvider client={client}>
              <CachePersistorProvider>
                <SafeAreaProvider>
                  <ActionSheetProvider>
                    <SuggestionsProvider>
                      <FeedWeightingProvider>
                        <>
                          <Navigation />
                          <StatusBar style="dark" />
                        </>
                      </FeedWeightingProvider>
                    </SuggestionsProvider>
                  </ActionSheetProvider>
                </SafeAreaProvider>
              </CachePersistorProvider>
            </ApolloProvider>
          </DatabaseProvider>
        </LoginProvider>
      </GestureHandlerRootView>
    </KeyboardProvider>
  );
```

Error copied from Apple console on Macbook (iPhone connected by USB): 
```
runOperations: -[_UIKeyboardArbiter retrieveClientDebugInformationWithCompletion:] Failed to access <_UIKeyboardArbiterClientHandle: 0x28080e470; PID 14229: com.chirpbirding.birda-dev <<UIKBArbiterClientFocusContext: 0x283970640; contextID = 2f109a25; sceneIdentity = com.apple.frontboard.systemappservices::FBSceneManager:sceneID%3Acom.chirpbirding.birda-dev-default >>; hosting PIDs {(
)}; level 5.000000; active YES [wants YES]; suppression 0; iav 0.000000; on screen YES; isAcquiringFocus: NO> remote service: Error Domain=NSCocoaErrorDomain Code=4099 "The connection from pid 14229 on mach service named com.apple.UIKit.KeyboardManagement.hosted was invalidated: client is gone." UserInfo={NSDebugDescription=The connection from pid 14229 on mach service named com.apple.UIKit.KeyboardManagement.hosted was invalidated: client is gone.}
```

**To Reproduce**
1. Go to a form with an image picker and text input
2. Press the text input and type something - verify that it works correctly and doesn't crash
3. Press the image picker. Picker opens
4. Choose an image or press cancel (doesn't matter which). Returns to form
5. Press text input. App crashes and closes immediately to the device desktop

**Expected behavior**
1. Go to a form with an image picker and text input
2. Press the text input and type something - verify that it works correctly and doesn't crash
3. Press the image picker. Picker opens
4. Choose an image or press cancel (doesn't matter which). Returns to form
5. Press text input and type something - it works correctly and doesn't crash

**Smartphone (please complete the following information):**
 - Desktop OS: [e.g. Windows 10, MacOS 10.15.5]
 - Device: iPhone X
 - OS: IOS 16.4.1
 - RN version: 0.71.6
 - RN architecture: old
 - JS engine: Hermes
 - Library version: 1.5.3 (also tried 1.4.4 to no avail)

**Comment by shedworth** [comment]
Still happening unfortunately :(

**Comment by kirillzyusko** [comment]
@shedworth that's strange... I added `react-native-image-picker` and added a button to open `image-picker`. It produced crash straight away after `TextInput` gets focused, but with this change it works fine 🤷‍♂️ 

I may suppose the problem is somewhere else, but in order to understand that I need to have a reproducible example 🙏

**Comment by kirillzyusko** [comment]
GitHub automatically closed this issue since PR that had a reference to this issue was merged.

I've published `1.5.4` to npm-registry. Hopefully the issue is resolved, but in any case feel free to re-open this issue or feel free to create a new one 🙂

**Comment by kirillzyusko** [comment]
@shedworth the similar problem was reported in https://github.com/kirillzyusko/react-native-keyboard-controller/issues/175

The reason why it couldn't be reproducible on my setup is the fact that I've used only iOS 16.2/16.3. However you were testing iOS 16.4

https://github.com/kirillzyusko/react-native-keyboard-controller/issues/175 was fixed and I published a new `1.5.7` version on npm. I re-tested my example (with image-picker and `1.5.7` version) and it works like a charm on iOS 16.4 too 🙂 

So feel free to re-test it too 😎 Hopefully the problem has fully gone 🤞
